app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftMediaAdapter.kt

```diff
@@ -42,24 +42,34 @@ class DraftMediaAdapter(
 ) {
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): DraftMediaViewHolder {
-        return DraftMediaViewHolder(AppCompatImageView(parent.context))
+        return DraftMediaViewHolder(MediaPreviewImageView(parent.context))
     }
 
     override fun onBindViewHolder(holder: DraftMediaViewHolder, position: Int) {
         getItem(position)?.let { attachment ->
             if (attachment.type == DraftAttachment.Type.AUDIO) {
+                holder.imageView.clearFocus()
                 holder.imageView.setImageResource(R.drawable.ic_music_box_preview_24dp)
             } else {
-                Glide.with(holder.itemView.context)
```

Comment:
Long term observation: This clause has (and had before I started changing it) significant code duplication VS MediaPreviewAdapter. Maybe it's worth a refactor here